src/index.js

```diff
@@ -27,72 +26,16 @@ cosmiconfig('lint-staged', {
   .then(result => {
     // result.config is the parsed configuration object
     // result.filepath is the path to the config file that was found
-    const config = result.config
-
-    const verbose = config.verbose
-    // Output config in verbose mode
-    if (verbose) console.log(config)
-    const concurrent = readConfigOption(config, 'concurrent', true)
-    const renderer = verbose ? 'verbose' : 'update'
-    const gitDir = config.gitDir ? path.resolve(config.gitDir) : process.cwd()
-    sgf.cwd = gitDir
-
-    sgf('ACM', (err, files) => {
-      if (err) {
-        console.error(err)
+    runAll(packageJson, result.config)
```

Comment:
We only need `packageJson.scripts` so how about passing only that?

Reply:
Yeah, I though about this as well but this already. Will take a look.